lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,189 @@
+ @use '../../scss-shared/nx-text-helpers';
+
+@keyframes slide-in {
+  0% {
+    transform: translateX(0);
+    opacity: 0;
+  }
+
+  // This is required to fix jittering in Firefox and Safari
+  // and not animating at all in Chrome.
+  1% {
+    transform: translateX(100%);
+    opacity: 0;
+  }
+
+  100% {
+    transform: translateX(0);
+    opacity: 1;
+  }
+}
+
+@keyframes slide-out {
+  0% {
+    transform: translateX(0);
+    opacity: 1;
+  }
+  100% {
+    transform: translateX(100%);
+    opacity: 0;
+  }
+}
+
+.nx-drawer {
+  // Undo styles that chrome applies to <dialog> elements
+  border: initial;
+  height: initial;
+  left: initial;
+  margin: initial;
+  max-height: initial;
+  max-width: initial;
+  padding: initial;
+  width: initial;
+
+  // Prevent brief flash of scrollbar as tooltips disappear
+  overflow: hidden;
+
+  &::backdrop {
+    // Chrome tries to add its own subtle mask here which doubles up with our own
+    display: none;
+    pointer-events: auto;
+  }
+
+  grid-area: content;
+  box-sizing: border-box;
+  position: relative;
+  background-color: transparent;
+  color: var(--nx-swatch-grey-30);
+  justify-self: end;
+  width: 100%;
+  pointer-events: none;
+  z-index: var(--nx-z-index-drawer);
+}
+
+.nx-drawer__animation-wrapper {
+  display: flex;
+  height: 100%;
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  // Prevents flickering
+  backface-visibility: hidden;
+  animation: slide-in 200ms ease-out;
+
+  &--close {
+    animation: slide-out 180ms ease-in;
+  }
+
+  @media (prefers-reduced-motion) {
+    &, &--close {
+      // We are setting the duration to 1ms
+      // because 0 did not trigger the animationEnd event.
+      animation-duration: 1ms;
+    }
+  }
+}
+
+.nx-drawer__panel {
+  box-sizing: border-box;
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  pointer-events: auto;
+  background-color: var(--nx-swatch-white);
+  position: relative;
+  padding: var(--nx-spacing-6x);
+  opacity: 1;
+  min-height: 0;
+  width: 520px;
+
+  &--narrow {
+    width: 348px;
+  }
+
+  &::after {
+    content: " ";
+
+    box-shadow: -2px 0px 50px rgba(128, 128, 128, 0.5);
+    height: calc(100% - var(--nx-spacing-16x));
+    width: 100%;
+    position: absolute;
+    left: 0;
+    top: var(--nx-spacing-8x);
+    z-index: -1;
+  }
+
+  .nx-form {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    height: 100%;
+    min-height: 0;
+  }
+
+  .nx-drawer-content {
+    --nx-drawer-padding: var(--nx-spacing-6x);
+
+    @include nx-container-helpers.container-vertical;
+
+    overflow: auto;
+    padding: var(--nx-spacing-6x);
+    height: 100%;
+    margin-right: calc(0px - var(--nx-drawer-padding));
+    margin-left: calc(0px - var(--nx-drawer-padding));
+    margin-bottom: calc(0px - var(--nx-drawer-padding));
+  }
+
+  .nx-footer {
+    @include nx-container-helpers.container-vertical;
+
+    border-top: 1px solid var(--nx-swatch-indigo-90);
```

Comment:
Isn't this redundant?

Reply:
> Isn't this redundant?

Fixed here: 8295ed1b6075714f87857196dc4be927fa0f9bd1